sql/core/src/main/scala/org/apache/spark/sql/execution/SubqueryBroadcastExec.scala

```diff
@@ -110,5 +115,6 @@ case class SubqueryBroadcastExec(
 
 object SubqueryBroadcastExec {
   private[execution] val executionContext = ExecutionContext.fromExecutorService(
-    ThreadUtils.newDaemonCachedThreadPool("dynamicpruning", 16))
+    ThreadUtils.newDaemonCachedThreadPool("dynamic-pruning",
+      SQLConf.get.getConf(StaticSQLConf.SUBQUERY_MAX_THREAD_THRESHOLD)))
```

Comment:
shall we use `BROADCAST_EXCHANGE_MAX_THREAD_THRESHOLD`? Physically it's a broadcast.

Reply:
yes we should use `BROADCAST_EXCHANGE_MAX_THREAD_THRESHOLD`, thanks